client/src/app/components/signup-form/signup-form.component.scss

```diff
@@ -35,5 +35,9 @@
       font-size: 26px;
       color: $mt-white;
     }
+    ::ng-deep .mat-form-field-flex{
+      background-color: white;
+      border-radius: 5px;
+    }
```

Comment:
Since this is a bug fix that should affect all input boxes and not only the ones in signup-form, this class should be moved to 'themes.scss'. But the fix works great!